Flow for sending multiple collectibles

vkjr opened the pull request:
fixes #19141

### Summary

Added flow for sending multiple collectibles
[Figma designs](https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=10771-86499&mode=design&t=4cIir16oeX5ckoNb-4)

https://github.com/status-im/status-mobile/assets/5786310/a35cdbbe-8a84-4695-becf-65ef472671ba


### Review notes
Some components required small adjustments:
- expand-collectible didn't need `:flex 1` because that breaks layout
- network-tags now has a `:container-style` property
- amount-input no longer controls its input, it is set via property. This component never used before and was designed like it is used with the default keyboard. But since we use it only with our own keyboard component, we should control value externally 


status: ready

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :x: | c3e7bde6 | [#2](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-19386/2/) | 2024-03-25 12:39:32 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-19386/2/consoleText) |

ulisesmac reviewed (approved):
Thanks for the PR! looks good

briansztamfater commented on src/status_im/contexts/wallet/collectible/utils.cljs:
```diff
@@ -0,0 +1,9 @@
+(ns status-im.contexts.wallet.collectible.utils)
+
+(defn collectibles-count
```
Maybe a better name for the function would be `collectible-balance`

briansztamfater commented on src/status_im/contexts/wallet/send/select_collectible_amount/view.cljs:
```diff
@@ -0,0 +1,67 @@
+       :square?         true
+       :container-style style/collectible-container}]
+     [quo/network-tags
+      {:title           (str (i18n/label :t/max) ": " balance)
```
Consider passing `balance` as a parameter to the i18n label. It will be easier to translate the string for RTL languages then.

briansztamfater commented on src/status_im/contexts/wallet/send/select_asset/view.cljs:
```diff
@@ -34,9 +35,15 @@
-      :on-collectible-press #(rf/dispatch [:wallet/send-select-collectible
-                                           {:collectible %
-                                            :stack-id    :screen/wallet.select-asset}])}]))
+      :on-collectible-press #(let [collectibles-count (utils/collectibles-count %)]
+                               (if (> collectibles-count 1)
+                                 (rf/dispatch [:wallet/select-collectibles-amount
+                                               {:collectible %
+                                                :stack-id    :screen/wallet.select-asset}])
+                                 (rf/dispatch [:wallet/send-collectibles-amount
+                                               {:collectible %
+                                                :amount      1
+                                                :stack-id    :screen/wallet.select-asset}])))}]))
```
Navigation should be handled by the wizard config. Check #19123

vkjr replied:
@briansztamfater, thanks, will check it!

briansztamfater commented on src/status_im/contexts/wallet/send/events.cljs:
```diff
@@ -101,16 +101,24 @@
+    :fx [[:dispatch [:wallet/get-suggested-routes {:amount amount}]]
          [:navigate-to-within-stack [:screen/wallet.transaction-confirmation stack-id]]]}))
 
+(rf/reg-event-fx :wallet/select-collectibles-amount
```
Event name sounds confusing to me. `:wallet/send-select-collectible` would be better IMO, as we are not handling any `amount` here.

Also I think these two events are very similar, we can have `:wallet/send-select-collectible`, if token balance is greater than 1, don't assoc-in the amount and navigate to the next screen (~handled by the wizard event~). Otherwise assoc-in amount with 1 and dispatch get-suggested-routes with amount 1 as well.

Then another simpler event `:wallet/send-collectible-set-amount` for collectibles with balances greater than 1, which should just store the amount on app-db, dispatch get-suggested-routes with the selected amount, and navigate to the following screen.

briansztamfater commented on src/status_im/contexts/wallet/send/events.cljs:
```diff
@@ -101,16 +101,24 @@
+            (update-in [:wallet :ui :send] dissoc :token)
+            (assoc-in [:wallet :ui :send :collectible] collectible)
+            (assoc-in [:wallet :ui :send :tx-type] :collectible))
+    :fx [[:navigate-to-within-stack [:screen/wallet.select-collectible-amount stack-id]]]}))
```
Also we should use `:wallet/wizard-navigate-forward` and let the wizard logic find the next screen for the flow depending on the defined conditions

J-Son89 replied:
hmm, it could depend on how much work is needed on the other screens to add that code though? perhaps that is better added in a follow up issue/pr? 🤔

J-Son89 replied:
we also have to add the wizard for the other parts of the collectibles sending flow. This pr is only adding 1 screen to a pre existing set of navigations.

briansztamfater replied:
We can do this in a follow up issue, sure 👍 
Do we have an issue for that? Also would be nice to add docs or guidelines for using the wizard mechanism.

briansztamfater replied:
Still https://github.com/status-im/status-mobile/pull/19386#discussion_r1537829259 is valid just that we can keep `:navigate-to-within-stack` (we should dispatch the event though, not use the effect).

J-Son89 replied:
@ulisesmac - is there an issue for the collectibles/bridging wizard stuff? I think we discussed last week.

Also adding a doc is a good idea. We can add a read me to the wallet maybe?

ulisesmac replied:
@J-Son89 I haven't added issues for it, since it was recently merged and I haven't started with it yet.

Edit:
Here's the issue
- https://github.com/status-im/status-mobile/issues/19410

briansztamfater reviewed (changes requested):
PR looks good in general, requested changes because we should rely on the wizard logic to navigate within the flow instead of hardcoding navigation. Added some suggestions on how we could refactor to use the wizard.

J-Son89 commented:
the video you attached has schema errors in it, can we address them? 🙏

J-Son89 commented on src/quo/components/profile/expanded_collectible/view.cljs:
```diff
@@ -55,7 +55,7 @@
                      {:label   (i18n/label :t/cant-fetch-info)
                       :counter counter
                       :theme   theme}]
-       [rn/view {:style {:flex 1}}
```
hmm, are you sure about this? The component looked right to me in the preview screen. 
Can you clear up why you did this? 🤔

vkjr replied:
@J-Son89, there is no reason for `expanded-collectible` to take all available space, space of the image should be enough.
I ran into a problem when placed components on the screen. Components order:
```
[quo/expanded-collectible]
[quo/network-tags]
[quo/amount-input]
...
```
Since expanded-collectible tried to occupy as much space as possible, it was pushing away `network-tags` making it impossible to maintain correct margin between `expanded-collectible` and `network-tags`

J-Son89 replied:
Thanks @vjkr, the thing is if the image is portrait it should respect that aspect ratio and the same for landscape. As long as that's still maintained then sounds good.

Thanks for the very clear explanation! 🙏

vkjr replied:
I double-checked with preview screen of `expanded-collectible` and its behaviour is intact

J-Son89 replied:
Perfect, thanks!

J-Son89 commented on src/status_im/contexts/wallet/send/select_collectible_amount/view.cljs:
```diff
@@ -0,0 +1,67 @@
+        balance               (utils/collectibles-count collectible)
+        preview-uri           (get-in collectible [:preview-url :uri])
+        incorrect-value?      (or (< value 1) (> value balance))]
+    [rn/safe-area-view
```
hmm is this how we normally handle safe-area 🤔 
do we not set this in the navigation screens file? 
cc @OmarBasem

OmarBasem replied:
Yes we should set in the navigation options: `:insets     {:top false}`

J-Son89 commented on src/quo/components/wallet/amount_input/view.cljs:
```diff
@@ -21,60 +20,34 @@
+     :on-press            on-dec-press
+     :disabled?           (>= min-value value)}]
+   [rn/view {:style style/input-container}
+    [text/text
```
🙌

J-Son89 commented on src/quo/components/wallet/amount_input/schema.cljs:
```diff
@@ -1,21 +1,16 @@
       [:max-value {:optional true} [:maybe :int]]
-      [:return-key-type {:optional true} [:maybe return-key-types]]
-      [:init-value {:optional true} [:maybe :int]]]]]
+      [:value [:maybe :int]]]]]
```
👍

vkjr commented:
> the video you attached has schema errors in it, can we address them? 🙏

@J-Son89, there shouldn't be schema errors in my changes, but I will check what happens around)

J-Son89 reviewed (approved):
Imo we should leave updating navigation mechanism to a separate issue/ pr solely focused on that 👍

Pr looks good anyway, nice one @vkjr! 🚀

vkjr commented:
@J-Son89, @briansztamfater 
Addressed review notes and added [an issue](https://github.com/status-im/status-mobile/issues/19405) for navigation change. Will update events within the scope of that issue.

vkjr committed:
Flow for sending multiple collectibles

status-im-auto commented:
## 94% of end-end tests have passed
```
Total executed tests: 48
Failed tests: 2
Expected to fail tests: 1
Passed tests: 45
```
```
IDs of failed tests: 702775,704613 
```
```
IDs of expected to fail tests: 703503 
```
<h3>Failed tests (2)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=develop&PR_ID=19386&TR_CASE_IDS=702775,704613">Rerun failed tests</a></li><br/><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511666">test_links_deep_links</a>, id: 702775 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `BrowserTab` by `accessibility id`: `browser-stack-tab` </div><div>Device 1: Tap on found: BrowserTab </div></blockquote></p>

```
critical/test_deep_and_universal_links.py:112: in test_links_deep_links
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community 'Status mobile QA community max' was not requested to join by the deep link status.app://c/G8EAAMR_fz8tsCQ-aR2QrCS5sVAvvzc_N3mAA-En_Zxy4JA3j7Dl1A50Pd4DbooQOMbWf7E1_4wipgDyGe8XZEappDn-Qomf9l_xyXhSYBuSQic8InCEUBSRGR0oixSTh3iw5ZCxzkGSI95Iyu1EBpcIlFOEMPHpKUBIdkkoKBJglMDKko8O8dBvBtIYncOA8mwztwLpx3C0rK_u59PldFuXe4cx#zQ3shwQnEfMtcXpHXF4qJPyCGgw2F18N3nbGzYbzsVHnMq4yK
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cfe4d0274dbb42648e9efa3c976d8c0c?auth=598a8a5dadf1c8885502566beda1d1f5#194">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511706">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `xpath`: `//*[@text="open community"]` </div><div>Device 1: Tap on found: Button </div></blockquote></p>

```
critical/test_deep_and_universal_links.py:69: in test_links_open_universal_links_from_chat
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community 'Status mobile QA community max' was not requested to join by the url https://status.app/c/G8EAAMR_fz8tsCQ-aR2QrCS5sVAvvzc_N3mAA-En_Zxy4JA3j7Dl1A50Pd4DbooQOMbWf7E1_4wipgDyGe8XZEappDn-Qomf9l_xyXhSYBuSQic8InCEUBSRGR0oixSTh3iw5ZCxzkGSI95Iyu1EBpcIlFOEMPHpKUBIdkkoKBJglMDKko8O8dBvBtIYncOA8mwztwLpx3C0rK_u59PldFuXe4cx#zQ3shwQnEfMtcXpHXF4qJPyCGgw2F18N3nbGzYbzsVHnMq4yK
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cfe4d0274dbb42648e9efa3c976d8c0c?auth=598a8a5dadf1c8885502566beda1d1f5#55">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Expected to fail tests (1)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511704">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td>

```
Test is not run, e2e blocker  
```

<b>[[reason: [NOTRUN] Curated communities not loading, https://github.com/status-im/status-mobile/issues/17852]]</b><br/><br/></td></tr></tbody></table></details><h3>Passed tests (45)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511691">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d830554c3df34932a43e42801b3ac342?auth=a5c4850be5afd3a6066e13fb956e1181#182">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/46aba9f1b3e94c278a78ae0d3596dd95?auth=33a64909a81b97485b7a5c7e71b30491#125">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511690">test_navigation_jump_to</a>, id: 702936 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d830554c3df34932a43e42801b3ac342?auth=a5c4850be5afd3a6066e13fb956e1181#154">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/46aba9f1b3e94c278a78ae0d3596dd95?auth=33a64909a81b97485b7a5c7e71b30491#122">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511678">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6651cb378c914205ac87f2fa47bac2c7?auth=ebfdd9a1966148fd77b5cda8524513fe#296">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/84966ebb18544a378557155bd2a13ca1?auth=bc33d76117d3b2f6c5618eabdbb96100#287">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511689">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6651cb378c914205ac87f2fa47bac2c7?auth=ebfdd9a1966148fd77b5cda8524513fe#402">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/84966ebb18544a378557155bd2a13ca1?auth=bc33d76117d3b2f6c5618eabdbb96100#353">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511695">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6651cb378c914205ac87f2fa47bac2c7?auth=ebfdd9a1966148fd77b5cda8524513fe#563">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/84966ebb18544a378557155bd2a13ca1?auth=bc33d76117d3b2f6c5618eabdbb96100#432">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511681">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6651cb378c914205ac87f2fa47bac2c7?auth=ebfdd9a1966148fd77b5cda8524513fe#320">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/84966ebb18544a378557155bd2a13ca1?auth=bc33d76117d3b2f6c5618eabdbb96100#302">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511679">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6651cb378c914205ac87f2fa47bac2c7?auth=ebfdd9a1966148fd77b5cda8524513fe#365">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/84966ebb18544a378557155bd2a13ca1?auth=bc33d76117d3b2f6c5618eabdbb96100#343">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511677">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6651cb378c914205ac87f2fa47bac2c7?auth=ebfdd9a1966148fd77b5cda8524513fe#206">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/84966ebb18544a378557155bd2a13ca1?auth=bc33d76117d3b2f6c5618eabdbb96100#138">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511676">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6651cb378c914205ac87f2fa47bac2c7?auth=ebfdd9a1966148fd77b5cda8524513fe#164">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/84966ebb18544a378557155bd2a13ca1?auth=bc33d76117d3b2f6c5618eabdbb96100#109">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511707">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6651cb378c914205ac87f2fa47bac2c7?auth=ebfdd9a1966148fd77b5cda8524513fe#602">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/84966ebb18544a378557155bd2a13ca1?auth=bc33d76117d3b2f6c5618eabdbb96100#462">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511687">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6651cb378c914205ac87f2fa47bac2c7?auth=ebfdd9a1966148fd77b5cda8524513fe#248">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/84966ebb18544a378557155bd2a13ca1?auth=bc33d76117d3b2f6c5618eabdbb96100#231">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511680">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6651cb378c914205ac87f2fa47bac2c7?auth=ebfdd9a1966148fd77b5cda8524513fe#182">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/84966ebb18544a378557155bd2a13ca1?auth=bc33d76117d3b2f6c5618eabdbb96100#126">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511697">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6651cb378c914205ac87f2fa47bac2c7?auth=ebfdd9a1966148fd77b5cda8524513fe#223">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/84966ebb18544a378557155bd2a13ca1?auth=bc33d76117d3b2f6c5618eabdbb96100#167">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511682">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cd4c481a78544c5b9e7c791e3d43c172?auth=3570305261b83e81da4f5a41af9fd469#335">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d97972a5b5cf4b3fa0c0b02ff35861cf?auth=135df2c91410b13bdd51a5b00d9ccfc2#301">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511671">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cd4c481a78544c5b9e7c791e3d43c172?auth=3570305261b83e81da4f5a41af9fd469#141">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d97972a5b5cf4b3fa0c0b02ff35861cf?auth=135df2c91410b13bdd51a5b00d9ccfc2#112">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511674">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cd4c481a78544c5b9e7c791e3d43c172?auth=3570305261b83e81da4f5a41af9fd469#197">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d97972a5b5cf4b3fa0c0b02ff35861cf?auth=135df2c91410b13bdd51a5b00d9ccfc2#177">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511692">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cd4c481a78544c5b9e7c791e3d43c172?auth=3570305261b83e81da4f5a41af9fd469#338">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d97972a5b5cf4b3fa0c0b02ff35861cf?auth=135df2c91410b13bdd51a5b00d9ccfc2#332">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511705">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cd4c481a78544c5b9e7c791e3d43c172?auth=3570305261b83e81da4f5a41af9fd469#541">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d97972a5b5cf4b3fa0c0b02ff35861cf?auth=135df2c91410b13bdd51a5b00d9ccfc2#515">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511664">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f563258fb8944f4ba996a52fb837a255?auth=634f62d4712e0e83f829ef1044ecbeff#79">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511683">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f563258fb8944f4ba996a52fb837a255?auth=634f62d4712e0e83f829ef1044ecbeff#55">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511696">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f563258fb8944f4ba996a52fb837a255?auth=634f62d4712e0e83f829ef1044ecbeff#252">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511688">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f563258fb8944f4ba996a52fb837a255?auth=634f62d4712e0e83f829ef1044ecbeff#108">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511700">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f563258fb8944f4ba996a52fb837a255?auth=634f62d4712e0e83f829ef1044ecbeff#127">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511703">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8c198239242d4f03952936962f4f9cc7?auth=21bac0acd520331ee0fa01151d2a22ef#108">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4b74025cd69d4f858b09a7fae4fdfc96?auth=d26eee716b01b16d1b54aa680f40d297#115">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511669">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8c198239242d4f03952936962f4f9cc7?auth=21bac0acd520331ee0fa01151d2a22ef#94">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4b74025cd69d4f858b09a7fae4fdfc96?auth=d26eee716b01b16d1b54aa680f40d297#87">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511670">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8c198239242d4f03952936962f4f9cc7?auth=21bac0acd520331ee0fa01151d2a22ef#182">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4b74025cd69d4f858b09a7fae4fdfc96?auth=d26eee716b01b16d1b54aa680f40d297#128">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511698">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/60aad290df3046ad9bb85f034dc11f1a?auth=8bc5cf93aa7217b7c2ff4422619351aa#178">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/160c67c64ef44f65a1af0cb87cf65b93?auth=e423822deb13cae6a7b664ecb222ef1b#109">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b177671f90884c37a81e2c12af7ae1cb?auth=f462cbb1ac3a713fae8425c06f7eccea#89">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511672">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/60aad290df3046ad9bb85f034dc11f1a?auth=8bc5cf93aa7217b7c2ff4422619351aa#159">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/160c67c64ef44f65a1af0cb87cf65b93?auth=e423822deb13cae6a7b664ecb222ef1b#81">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b177671f90884c37a81e2c12af7ae1cb?auth=f462cbb1ac3a713fae8425c06f7eccea#81">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511673">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/60aad290df3046ad9bb85f034dc11f1a?auth=8bc5cf93aa7217b7c2ff4422619351aa#327">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/160c67c64ef44f65a1af0cb87cf65b93?auth=e423822deb13cae6a7b664ecb222ef1b#218">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b177671f90884c37a81e2c12af7ae1cb?auth=f462cbb1ac3a713fae8425c06f7eccea#207">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511662">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/60aad290df3046ad9bb85f034dc11f1a?auth=8bc5cf93aa7217b7c2ff4422619351aa#364">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/160c67c64ef44f65a1af0cb87cf65b93?auth=e423822deb13cae6a7b664ecb222ef1b#244">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b177671f90884c37a81e2c12af7ae1cb?auth=f462cbb1ac3a713fae8425c06f7eccea#241">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511699">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/60aad290df3046ad9bb85f034dc11f1a?auth=8bc5cf93aa7217b7c2ff4422619351aa#283">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/160c67c64ef44f65a1af0cb87cf65b93?auth=e423822deb13cae6a7b664ecb222ef1b#179">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b177671f90884c37a81e2c12af7ae1cb?auth=f462cbb1ac3a713fae8425c06f7eccea#148">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511702">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/60aad290df3046ad9bb85f034dc11f1a?auth=8bc5cf93aa7217b7c2ff4422619351aa#488">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/160c67c64ef44f65a1af0cb87cf65b93?auth=e423822deb13cae6a7b664ecb222ef1b#290">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b177671f90884c37a81e2c12af7ae1cb?auth=f462cbb1ac3a713fae8425c06f7eccea#259">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511685">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d21b17dd62934f85859b85ed72fbe568?auth=c8bc4adb28340d66811434519eb73dbb#101">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/03132fea793845efaae8704d72a474e0?auth=472ccce8f80abb187e2bee90c9dee3b5#128">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511684">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d21b17dd62934f85859b85ed72fbe568?auth=c8bc4adb28340d66811434519eb73dbb#59">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/03132fea793845efaae8704d72a474e0?auth=472ccce8f80abb187e2bee90c9dee3b5#66">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511667">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d21b17dd62934f85859b85ed72fbe568?auth=c8bc4adb28340d66811434519eb73dbb#136">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/03132fea793845efaae8704d72a474e0?auth=472ccce8f80abb187e2bee90c9dee3b5#224">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511686">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6e2e125729ce44d9ad6f20b8c3d58b97?auth=98bac1b11357f6873501cd626c5d825b#469">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0cfc44bbcfbb4933b9dd3e4539313b12?auth=c770c05b42813ab57e412d9e7041e625#298">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511660">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6e2e125729ce44d9ad6f20b8c3d58b97?auth=98bac1b11357f6873501cd626c5d825b#98">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0cfc44bbcfbb4933b9dd3e4539313b12?auth=c770c05b42813ab57e412d9e7041e625#87">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511665">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6e2e125729ce44d9ad6f20b8c3d58b97?auth=98bac1b11357f6873501cd626c5d825b#352">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0cfc44bbcfbb4933b9dd3e4539313b12?auth=c770c05b42813ab57e412d9e7041e625#209">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511661">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6e2e125729ce44d9ad6f20b8c3d58b97?auth=98bac1b11357f6873501cd626c5d825b#214">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0cfc44bbcfbb4933b9dd3e4539313b12?auth=c770c05b42813ab57e412d9e7041e625#169">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511663">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6e2e125729ce44d9ad6f20b8c3d58b97?auth=98bac1b11357f6873501cd626c5d825b#558">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0cfc44bbcfbb4933b9dd3e4539313b12?auth=c770c05b42813ab57e412d9e7041e625#401">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511675">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6e2e125729ce44d9ad6f20b8c3d58b97?auth=98bac1b11357f6873501cd626c5d825b#414">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0cfc44bbcfbb4933b9dd3e4539313b12?auth=c770c05b42813ab57e412d9e7041e625#263">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511668">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6e2e125729ce44d9ad6f20b8c3d58b97?auth=98bac1b11357f6873501cd626c5d825b#152">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0cfc44bbcfbb4933b9dd3e4539313b12?auth=c770c05b42813ab57e412d9e7041e625#124">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511701">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6e2e125729ce44d9ad6f20b8c3d58b97?auth=98bac1b11357f6873501cd626c5d825b#497">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0cfc44bbcfbb4933b9dd3e4539313b12?auth=c770c05b42813ab57e412d9e7041e625#341">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511694">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/919c74c789b047749aeb4a75bdb0dbdd?auth=5179413dbc4002c80455af423cde63d8#178">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d814e562a663451394bd7680b149cb57?auth=b20320a169840dce9f4b3a6ac786fa1b#119">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3511693">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/919c74c789b047749aeb4a75bdb0dbdd?auth=5179413dbc4002c80455af423cde63d8#130">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d814e562a663451394bd7680b149cb57?auth=b20320a169840dce9f4b3a6ac786fa1b#103">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>
